pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/renderer/CompositeValueProvider.java

```diff
@@ -1,29 +1,6 @@
 package org.broadinstitute.dsm.model.elastic.export.tabular.renderer;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.stream.Collectors;
-
-import org.apache.commons.lang3.StringUtils;
-
 /** handles questions with lists of lists for answers */
 public class CompositeValueProvider extends TextValueProvider {
```

Comment:
do we still need the CompositeValueProvider if it only extends TextValueProvider

Reply:
Good point, I'll keep it for now, since I suspect we might change course again on how address questions are displayed (e.g. comma-delimited, as a list, etc...